usr/share/rear/prep/BACULA/default/500_check_BACULA_bconsole_results.sh

```diff
@@ -29,19 +27,16 @@ fi
 # and that the director can connect to the file daemon on this system.
 # "Connecting to Director 'director_name-fd:9101'"
 # "Connecting to Client 'bacula_client_name-fd at FQDN:9102"
-BACULA_CLIENT=`grep $(hostname -s) /etc/bacula/bacula-fd.conf | grep "\-fd" | awk '{print $3}' | sed -e "s/-fd//g"`
-[ "${BACULA_CLIENT}" ]
-StopIfError "Client $(hostname -s) not defined in /etc/bacula/bacula-fd.conf"
+BACULA_CLIENT=$(grep $(hostname -s) $BACULA_CONF_DIR/bacula-fd.conf | grep "\-fd" | awk '{print $3}' | sed -e "s/-fd//g")
+[ "${BACULA_CLIENT}" ] || Error "Client $(hostname -s) not defined in $BACULA_CONF_DIR/bacula-fd.conf"
 
-BACULA_RESULT=( `echo -e " status client=${BACULA_CLIENT}-fd" | bconsole |grep Connect ` )
+BACULA_RESULT=( $(echo -e " status client=${BACULA_CLIENT}-fd" | bconsole | grep Connect) )
```

Comment:
As far as I see this is the actual test if the BACULA service is accessible
and I assume its result is the only one that is actually meaningful.
If I am right the 'ping' test above should be dropped as it is not meaningful
regarding what is actually tested but it could lead to a false error exit.